src/components/tutorialSelector/index.tsx

```diff
@@ -105,10 +115,9 @@ const TutorialSelector: React.FC = (): JSX.Element => {
           <SingleTutorial
             key={tutorialName + index.toString()}
             onClick={(): void => {
-              setTutorialStep(1);
-              setTutorialName(value.tutorial_title);
-              setSelectorDisplay(false);
-              setTutorialDisplay(true);
+              handleTutorialStep(tutStep);
+              handleCurrentTutorial(value.tutorial_title);
+              handleDisplays();
```

Comment:
This is a better direction but I think you could take it a step further, combine these all into a single function, and get rid of the error function by just calling onClick={newFunction} without needing the anonymous function. And then your new bigger functions just add a few comments on what state each one changes (i.e. changes global state vs changes localStorage state).

Reply:
`onClick={(): void => handleClick(value.tutorial_title)}`

Since I'm mapping value.tutorial_title to set the title local storage and global state, do you want me to pass value.tutorial_title like this or as props?

Reply:
This works 👍